protocol-designer/src/pages/Settings/index.tsx

```diff
@@ -0,0 +1,265 @@
+            padding={`${SPACING.spacing16} ${SPACING.spacing24}`}
+            justifyContent={JUSTIFY_SPACE_BETWEEN}
+          >
+            <StyledText desktopStyle="bodyDefaultSemiBold">
+              {t('shared:pd_version')}
+            </StyledText>
+            <StyledText desktopStyle="bodyDefaultRegular">
+              {process.env.OT_PD_VERSION}
+            </StyledText>
+          </Flex>
+        </Flex>
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing8}>
+          <StyledText desktopStyle="bodyLargeSemiBold">
+            {t('shared:user_settings')}
+          </StyledText>
+          <Flex
+            borderRadius={BORDERS.borderRadius4}
+            backgroundColor={COLORS.grey10}
+            padding={`${SPACING.spacing16} ${SPACING.spacing24}`}
+            justifyContent={JUSTIFY_SPACE_BETWEEN}
+            alignItems={ALIGN_CENTER}
+          >
+            <Flex flexDirection={DIRECTION_COLUMN}>
+              <StyledText desktopStyle="bodyDefaultSemiBold">
+                {t('shared:hints')}
+              </StyledText>
+              <Flex color={COLORS.grey60}>
+                <StyledText desktopStyle="bodyDefaultRegular">
+                  {t('shared:reset_hints_and_tips')}
+                </StyledText>
+              </Flex>
+            </Flex>
+            <Btn
+              disabled={!canClearHintDismissals}
+              textDecoration={
+                canClearHintDismissals
+                  ? TYPOGRAPHY.textDecorationUnderline
+                  : FLEX_AUTO
+              }
+              onClick={() => dispatch(tutorialActions.clearAllHintDismissals())}
+            >
+              <StyledText desktopStyle="bodyDefaultRegular">
+                {canClearHintDismissals
+                  ? t('shared:reset_hints')
+                  : t('shared:no_hints_to_restore')}
+              </StyledText>
+            </Btn>
+          </Flex>
+        </Flex>
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing8}>
+          <StyledText desktopStyle="bodyLargeSemiBold">
+            {t('shared:privacy')}
+          </StyledText>
+          <Flex
+            borderRadius={BORDERS.borderRadius4}
+            backgroundColor={COLORS.grey10}
+            padding={`${SPACING.spacing16} ${SPACING.spacing24}`}
+            justifyContent={JUSTIFY_SPACE_BETWEEN}
+            alignItems={ALIGN_CENTER}
+            gridGap={SPACING.spacing80}
+          >
+            <Flex flexDirection={DIRECTION_COLUMN}>
+              <StyledText desktopStyle="bodyDefaultSemiBold">
+                {t('shared:shared_sessions')}
+              </StyledText>
+              <Flex color={COLORS.grey60}>
+                <StyledText desktopStyle="bodyDefaultRegular">
+                  {t('shared:we_are_improving')}
+                </StyledText>
+              </Flex>
+            </Flex>
+            <Btn
+              role="switch"
+              data-testid="analyticsToggle"
```

Comment:
probably we should add `aria-checked`

```suggestion
              ariaChecked={Boolean(hasOptedIn)}
              data-testid="analyticsToggle"
```